code/datums/tutorial/marine/hospital_corpsman_sandbox.dm

```diff
@@ -0,0 +1,679 @@
+	var/list/limbs = active_agent.limbs
+	var/amount_of_parts = round(rand(1, 6))
+
+	for(var/i in 1 to amount_of_parts)
+		var/obj/limb/selectedlimb = pick(limbs)
+		var/damageamount = (round(rand((40 * TUTORIAL_HM_INJURY_SEVERITY_BOOBOO), (50 * TUTORIAL_HM_INJURY_SEVERITY_BOOBOO))))
+		selectedlimb.take_damage(round((damageamount * damageamountsplit) / amount_of_parts), round((damageamount * (1 - damageamountsplit)) / amount_of_parts))
+		if((damageamount > 30) && (rand()) < (TUTORIAL_HM_INJURY_SEVERITY_BOOBOO / 10))
+			selectedlimb.fracture()
+
+	active_agent.updatehealth()
+	active_agent.UpdateDamageIcon()
+	active_agents |= active_agent
+	RegisterSignal(active_agent, COMSIG_HUMAN_TUTORIAL_HEALED, PROC_REF(make_agent_leave))
+	RegisterSignal(active_agent, COMSIG_HUMAN_SET_UNDEFIBBABLE, PROC_REF(make_agent_leave))
+
+/datum/tutorial/marine/hospital_corpsman_sandbox/proc/simulate_evac(datum/source, mob/living/carbon/human/target)
+
+	TUTORIAL_ATOM_FROM_TRACKING(/obj/structure/bed/medevac_stretcher/prop, medevacbed)
+
+	var/list/statusmessage = list()
+
+	for(var/datum/reagent/medical/chemical in target.reagents.reagent_list)
+		if(chemical.volume >= chemical.overdose_critical)
+			statusmessage |= "Critical [chemical.name] overdose detected"
+	for(var/datum/internal_organ/organ in target.internal_organs)
+		if(organ.damage >= organ.min_broken_damage)
+			if((locate(/datum/reagent/medical/peridaxon) in target.reagents.reagent_list) || (target.stat == DEAD))
+				statusmessage |= "Ruptured [organ.name] detected"
+			else
+				medevacbed.balloon_alert_to_viewers("Organ damage detected! Please stabilize patient with Peridaxon before transit.", null, DEFAULT_MESSAGE_RANGE, null, COLOR_RED)
+				playsound(medevacbed.loc, 'sound/machines/twobeep.ogg', 20)
+				return
+
+	if(tutorial_mob == target)
+		medevacbed.balloon_alert_to_viewers("Error! Unable to self-evacuate!", null, DEFAULT_MESSAGE_RANGE, null, COLOR_RED)
+		playsound(medevacbed.loc, 'sound/machines/twobeep.ogg', 20)
+		return
+	if(length(statusmessage) > 0)
+		medevacbed.balloon_alert_to_viewers("[pick(statusmessage)]! Evacuating patient!!", null, DEFAULT_MESSAGE_RANGE, null, LIGHT_COLOR_BLUE)
+		playsound(medevacbed.loc, pick(90;'sound/machines/ping.ogg',10;'sound/machines/juicer.ogg'), 20)
+		spawn(2.7 SECONDS)
+		flick("winched_stretcher", medevacbed)
+		make_agent_leave(target, TRUE)
+		medevacbed.update_icon()
+	else
+		medevacbed.balloon_alert_to_viewers("Error! Patient condition does not warrant evacuation!", null, DEFAULT_MESSAGE_RANGE, null, COLOR_RED)
+		playsound(medevacbed.loc, 'sound/machines/twobeep.ogg', 20)
+		return
+
+/datum/tutorial/marine/hospital_corpsman_sandbox/process(delta_time)
+
+	if((length(dragging_agents)) > 0)
+		move_dragging_agent()
+	if((length(active_agents)) > 0)
+		move_active_agents()
+
+/datum/tutorial/marine/hospital_corpsman_sandbox/proc/item_cleanup(obj/item/clothing/suit/storage/marine/medium/armor)
```

Comment:
Noticed during testing that if you accidentally put your scanner in a patient's armor(a fairly common mistake for new medics to make) it gets deleted with the armor. Add a bit to empty out the armor as it's deleted?